docs/features/startup_and_waits.md

```diff
@@ -1,5 +1,13 @@
 # Waiting for containers to start or be ready
 
+## Depending on another container
+
+When using multiple containers, it is possible that one container that you are starting relies on the other to be started before it should be started itself. An example of this might be a database that needs to be started before your application container can link to it. You can tell a container that it depends on another container by using the `dependsOn` method:
```

Comment:
```suggestion
Sometimes, a container relies on another container to be ready before it should start itself. An example of this might be a database that needs to be started before your application container can link to it. You can tell a container that it depends on another container by using the `dependsOn` method:
```

Reply:
Thanks! Had some issues with `./gradlew spotlessApply` but I think I figured it out.